snowfall/common.py

```diff
@@ -88,9 +88,11 @@ def load_checkpoint(
             src_key = '{}.{}'.format('module', key)
             dst_state_dict[key] = src_state_dict.pop(src_key)
         assert len(src_state_dict) == 0
-        model.load_state_dict(dst_state_dict)
+        model.load_state_dict(dst_state_dict, strict=False)
```

Comment:
@danpovey 
Adding `strict=False` should prevent PyTorch from complaining
about extra key `P_scores` in the checkpoints.